owncloudApp/src/main/java/com/owncloud/android/ui/activity/PassCodeActivity.java

```diff
@@ -342,9 +342,9 @@ private void processFullPassCode() {
                 savePassCodeAndExit();
 
             } else {
-                showErrorAndRestart(
-                        R.string.pass_code_mismatch, R.string.pass_code_configure_your_pass_code, View.VISIBLE
-                );
+                mPassCodeHdr.setText(R.string.pass_code_mismatch);
+                mPassCodeHdrExplanation.setVisibility(View.VISIBLE);
+                clearBoxes();
```

Comment:
`showErrorAndRestart` is used somewhere else. That means that some errors will be shown in a snack bar and we will have the same problem after this fix. Could we try to show the potential errors in the same way? What do you think?